public/index.html

```diff
@@ -21,7 +21,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Doppler | Reportes</title>
```

Comment:
Since this webapp will have more than reports, what do you think about hardcoding only `Doppler` in the HTML.

Besides that, optionally, you could change the title to `Doppler | Reportes` (with translation) in `componentDidMount` of `Reports`.

Reply:
Fixed